docs/api/rest.md

```diff
@@ -202,6 +202,7 @@ You can pass the following optional GET parameters:
 - `filter_entity_id=<entity_ids>` to filter on one or more entities - comma separated.
 - `end_time=<timestamp>` to choose the end of the period in URL encoded format (defaults to 1 day).
 - `minimal_response` to only return `last_changed` and `state` for states other than the first and last state (much faster).
+- `no_attributes` to avoid selecting attributes from the database (much faster).
```

Comment:
```suggestion
- `no_attributes` to skip returning attributes from the database (much faster).
```
If using this is much faster, would it make sense to reverse the option to only return the attributes when explicitly requested?

Reply:
Sadly we can't for backwards compatibility

Reply:
Ah, I see... How does this function in combination with the `minimal_response` described above? I'd assume it overrides this, but is it worth to document it explicitly? Also, does it make sense to document what "attributes" are in this context (I assume they are `state_attributes`?)?

Reply:
If functions the same with or without minimal response. No attributes on the states.

Reply:
> Also, does it make sense to document what "attributes" are in this context (I assume they are state_attributes?)?

There is only one attributes field in the response so I didn't think it was needed